### ISSUE TITLE
Feature: Community status metadata

### ISSUE THREAD

**Issue by sjn** [perlorg/www.pm.org#193]
Would it be meaningful to somehow distinguish defunct/hibernating groups from the ones that are active?

I haven't put much thought into this particular topic, but I guess CPANSec's attempt at a [list of project conditions](https://github.com/CPAN-Security/security.metacpan.org/blob/lifecycle/docs/foss-project-lifecycle.md) may be an inspiration?

**Comment by jhannah** [comment]
If you're aware of inactive groups, I'll deactivate them. I don't police the groups, but I deactivate sleeping ones when they're brought to my attention. Generally speaking if a group's website is down, or 1+ years out of date, and there's no mailing list activity then I deactivate them. I notify the group leader so they can correct the problem(s), at which time I reactivate them. e.g. #184

**Comment by sjn** [comment]
Yeah, "policing" is not what I'm talking about. 

I'm proposing adding some extra metadata fields so the people who are organising a group can share organisational matters with people in their area.

**Comment by jhannah** [comment]
Oh, ok, sure. What do you want to add? https://www.pm.org/faq/admin.html#xml

**Comment by sjn** [comment]
> Oh, ok, sure. What do you want to add? https://www.pm.org/faq/admin.html#xml

A but unsure, so I guess I'd love to start with a short exploration of the topic?

To get the exploration going, I'm wondering if it would be good to first distinguish between help that a group _needs_ from what it _wants_. "Needs" would be the kind of help that make the group continue or restart it's activities. "Wants" would cover typical tasks that would beneficial if someone stepped up and did.

For example,

* Need: Main organizers
* Need: Event co-organizers (maybe different roles: Social, Technical, Large)
* Want: SoMe/outreach contributors
* Want: Community manager/DevRel
* Want: OSPO & Recruiters Liaison

I guess there could be other tasks? I'd love to hear from others who have organized meetups or PM meetings.

**Comment by jhannah** [comment]
Don't let me dissuade you from whatever you want to do. Go for it! Extend the DTD as you like. 🙂

That said, my personal thoughts are that:
* Every inactive group needs 1+ main organizers. We don't need a fancier spec for that, it's already clear because the group is inactive.
* Every active group is, presumably, reaching out to their local group members about their internal needs, via whatever communication vectors they use. I don't see any value in the pm.org international website having complicated standardized data specifications for storing those needs.
* I could be wrong. 🙂

**Comment by ranguard** [comment]
That level of meta information really does feel like it's useful to each individual group - and that's what their mailing lists (or how ever they communicate) are for...

Putting it centrally - it will just be out of date before you begin!

**Comment by sjn** [comment]
>     * Every inactive group needs 1+ main organizers. We don't need a fancier spec for that, it's already clear because the group is inactive.

I'm unsure if it's clear _enough_. My experience is that most businesses that use Open Source don't really follow their communities close enough to realize this.

So for me, this suggestion is more about making "tribal" knowledge like this available for everyone, than the fanciness of the spec. :sweat_smile: 

>     * Every active group is, presumably, reaching out to their local group members about their internal needs, via whatever communication vectors they use. I don't see any value in the pm.org international website having complicated standardized data specifications for storing those needs.

I can't claim knowledge about how things are done across most of the Perl Mongers, but my impression about the few I've been exposed to, is that...

1. Groups usually lose activities gently and quietly, mostly because some core contributors or "doers" find something else/better to do.
2. Very few groups seem to have a active sense (not to speak of _plans_) around succession planning.
3. One of the _biggest_ drawbacks of this particular community is (I'm sad to point out) the notion that you profess about the usefulness of centralized resources. While this type of "anarchic" way of operating has many fantastic features (e.g. everyone is free to spend their efforts according to their own will, and all the good stuff that stems from this), it also makes cooperation and coordination so much harder that it becomes tedious to even try — and those who despite this attempt to create anything anyway, they have to struggle with an incredibly difficult "market" to reach out with their ideas and efforts.

With that said, my proposal isn't useful alone, but probably has to be one of several improvements to how this community operates. For example, I think it would make sense to explore ways of regularly reaching out to the group organizers with relevant reminders and news about the site.

>     * I could be wrong. 🙂

This is true for us all. :sweat_smile:

**Comment by sjn** [comment]
> That level of meta information really does feel like it's useful to each individual group - and that's what their mailing lists (or how ever they communicate) are for...
> 
> Putting it centrally - it will just be out of date before you begin!

Maybe? I don't think a mail produces the same communicative effect as a status page though, so I'm not sure I agree with you.

You do have a point though. If metadata like these are to become useful, they'll have to be relevant, "recent enough", and easy to update as they change.

To find out what fields are needed for this to become relevant is the topic of this ticket. :grin: 

Finding ways to keep the fields' contents "recent enough" is both a question related to the typical update-frequency and "staticness" of any given field, and therefore also a topic for this ticket? :smile: 

And with regard to ways to update the fields... First, we already have a really good way of doing it (PR's to this repo), but I'm guessing the reason people don't do it is more related to ignorance of the possibility of this, or perhaps due to lack of people who care enough to do it (assuming their ignorance has been addressed). In any case, I'm thinking that's a topic worth exploring in another ticket. :slightly_smiling_face:

**Comment by jhannah** [comment]
> I think it would make sense to explore ways of regularly reaching out to the group organizers with relevant reminders and news about the site.

One resource you might not be aware of: There is still a Group Leaders Mailman list, so the ~230 Group Leaders can all email each other en masse whenever they want. https://www.pm.org/faq/admin.html#Mailman No one has posted to that list since ~2018.

From inside what might be my blind spot, I don't see how Perl folk outside Omaha could help me relaunch Omaha.pm even if I wanted to do that. If I wanted to do that, I should gather people in Omaha interested in Perl. I'm not aware that there are any anymore. All the nerd events I attend, I'm known as the Perl guy, haven't found others.

(Not just Perl, this is also true of Omaha Python and Golang. I've been chatting on local nerd boards about (re-)starting a group for those languages too, have gathered zero interest.)

**Comment by oalders** [comment]
> nerd boards

Learned a new term today. 😅

**Comment by sjn** [comment]
> One resource you might not be aware of: There is still a Group Leaders Mailman list, so the ~230 Group Leaders can all email each other en masse whenever they want. https://www.pm.org/faq/admin.html#Mailman No one has posted to that list since ~2018.

Hm. I can't seem to log in there, and I'm getting a 500-error when accessing the "Unsubscibe or edit options" form on the [pm_groups](https://mail.pm.org/mailman/listinfo/pm_groups) list page. Are you in a position to investigate?

> From inside what might be my blind spot, I don't see how Perl folk outside Omaha could help me relaunch Omaha.pm even if I wanted to do that. If I wanted to do that, I should gather people in Omaha interested in Perl. I'm not aware that there are any anymore. All the nerd events I attend, I'm known as the Perl guy, haven't found others. (I found one Raku hobbyist once, which blew my mind, but they didn't want to attend any local meetup, and have since moved.)

Yeah, I guess your story may be a common one. Still, maybe it's worth shifting the perspective a little, in order to think anew on some of these issues?

For example:

* With the upcoming regulations around cybersecurity ([NIS2](https://security.metacpan.org/docs/readinglist.html#eu-and-eea-nis2), [CRA](https://security.metacpan.org/docs/readinglist.html#eu-and-eea--cra), [EO 14028](https://security.metacpan.org/docs/readinglist.html#usa--eo-14028) , there's going to be a need for people that can help get old Perl applications compliant – and to find these individuals, Perl Mongers can play a meaningful role – if these communities are known of, findable and responsive.
* Recruiting companies are still struggling finding Perl devs, and can _still_ be made easier if this is done _with_ community assistance, rather than how it's done now.
* The act of finding communities is the twin of the act of making communities findable – and this is where a resource like pm.org can help everyone.
* People move. A developer with some Open Source exposure might get a job in Omaha; How easily will they find Omaha.pm? This goes for the general case too – any other city/PM group. Maybe it's better to assume unawareness among the target demographics, and work from there?

My point is that "Open Source Community" is a topic that's going to become more and more relevant for businesses 

> (Not just Perl, this is also true of Omaha Python and Golang. I've been chatting on local nerd boards about (re-)starting a group for those languages too, have gathered zero interest.)

Ooof. I can relate with this too. There seems to be a lot of "bystanderism" going on, it seems. Or lack of interest in community building/care/sustainability. :disappointed:

**Comment by sjn** [comment]
Huh. Seems I was removed from pm_groups in 2009, and I didn't even notice :sweat_smile: 

But sure, that sounds like a good place to announce some updates and/or run a survey about community status metadata.

**Comment by jhannah** [comment]
Re: https://mail.pm.org/mailman/listinfo/pm_groups

> Hm. I can't seem to log in there, and I'm getting a 500-error when accessing the "Unsubscribe or edit options" form on the [pm_groups](https://mail.pm.org/mailman/listinfo/pm_groups) list page. Are you in a position to investigate?

Yup. Are you a group leader? I don't see `Nilsen` in our XML file of group leaders... That list is for group leaders. If you're leading a group I can add you (and fix our XML file?).

>  Recruiting companies are still struggling finding Perl devs, and can _still_ be made easier if this is done _with_ community assistance, rather than how it's done now.

They are? I'm looking for a job. Any links to Perl telecommuting jobs would be *great*!  ❤️

> The act of finding communities is the twin of the act of making communities findable – and this is where a resource like pm.org can help everyone.

That's what pm.org does, right? It shows anyone looking for a group where the active groups are?

> People move. A developer with some Open Source exposure might get a job in Omaha; How easily will they find Omaha.pm? This is also the case for any other city/PM group. Maybe it's better to assume unawareness among the target demographics, and work from there?

I mean... Can I at least have dibs on applying for the first Omaha Perl job before we open it up to global competition? 😉 They can pass on me, that's fine, but I'd at least like to apply. I hired a Perl dev in Omaha in 2003... he said that was super rare, even back then.

> Ooof. I can relate with this too. There seems to be a lot of "bystanderism" going on, it seems. Or a lack of interest in community building/care/sustainability in general? 😞

Broadly, there seems to be less interest in nerd meetups since covid. And broadly, Perl / Python / Golang appear to be less used, or used by people disinterested in physical meetups. (Which is fine. I get it. Jobs can be exhausting, maybe the last thing you want after 40 hours of coding is more coding / computer talk.)

Again, don't let me dissuade you from extending our `perl_mongers.dtd`. Go for it! I'd love to find out that I'm wrong. Let me know how I can help.

**Comment by jhannah** [comment]
Huh. I got an email that you sent an email to the list that I could moderate (approve and add you to the list), but then I logged into the interface and

> There are no pending requests.

Did you withdraw your pending post?

**Comment by sjn** [comment]
Seems https://www.pm.org/faq/admin.html already has a status attribute as part of the group elements. I can see the following ones:

* active
* inactive
* sleeping
* undef
* dead
* mlb
* leb
* gone
* unknown
* spam
* disbanded to make room for other groups -jhannah 20061203
* on hold
* Vetoed by Robert. See RT 57812.

Together with these, quite a few have XML comments associated with them.

So, for a first few fixes, I think it would be smart to to this..

1.  consolidate the status values into a limited set of enums that clearly communicate what the current state of the group is.
    * bring your ideas! :grin: 
3. introduce structured comments for noting an update's contributor name, timestamp and comment
    * e.g. `<changelog timestamp="2010.05.22" comment="Les reported that they become inactive as there was little participation" contributor="szabgab" />`)
    * this could also be used for providing a reason for a status change
4. introduce a new element for each group, that allows for a group leader (or support person, or other volunteer, maybe?) to communicate group needs (meaning – how they are looking to change, and what they need to do this), perhaps starting with my examples in https://github.com/perlorg/www.pm.org/issues/193#issuecomment-2282848817 ?

What do you guys think?

**Comment by sjn** [comment]
> > 1. consolidate the status values into a limited set of enums that clearly communicate what the current state of the group is.
> 
> That's fine, I can merge any such PR.

Could you also join in with the brainstorming? :sweat_smile: 
 
> > 2. introduce structured comments for noting an update's contributor name, timestamp and comment
> 
> Isn't the `git log` sufficient for those?

Maybe as a data source? Adding structured (but _not_ `<!-- -->`) comments to the XML itself would be with the purpose of exposing this new information somehow. For example, I'm thinking it would be useful to change the map pin of the group based on the status and needs it has, and use a hover/mouseover effect to show the latest comment describing what happened...

> > 3. introduce a new element for each group, that allows for a group leader (or support person, or other volunteer, maybe?) to communicate group needs (meaning – how they are looking to change, and what they need to do this), perhaps starting with my examples in [Feature: Community status metadata #193 (comment)](https://github.com/perlorg/www.pm.org/issues/193#issuecomment-2282848817) ?
> 
> Yup, feel free to extend our DTD however you want.

Part of what I'm proposing here is to use enumerators, which means I'll need some help figuring out what they should be. Wanna help? :smile:

**Comment by sjn** [comment]
Oh, and I guess some of the tooling in `bin/` may also need updating if the DTD/XML changes?

**Comment by sjn** [comment]
> Huh. I got a "pending subscriptions" email (Salve J. Nilsen) Tue Aug 13 02:46:56 2024, but I logged in and:
> 
> > There are no pending requests.
> 
> Did you want to be on the list?

I think Oslo.pm forgot to update this file after Marcus Ramberg stepped down. I guess I'm the current one, though we haven't had any activity since before Covid. I guess you could reset it to my old address?

I don't _need_ to be on the list. I was just curious about what was in the archives, but was blocked from looking at them as I wasn't a list member. I had forgotten that this one is only for group leaders.

(On a side note: I think groups should be allowed to assign more members, so they aren't forced into creating another burnout-prone SPOF due to "pm.org policy")

**Comment by sjn** [comment]
> > Recruiting companies are still struggling finding Perl devs, and can _still_ be made easier if this is done _with_ community assistance, rather than how it's done now.
> 
> They are? I'm looking for a job. Any links to Perl telecommuting jobs would be _great_! ❤️

Don't know how things are in the U.S., but yes, there's been a slump a while now. It's still possible to find stuff, and for anyone up for diving into the DevSecOps space, there'll be a bunch of opportunities showing up next year. :smile:

> > The act of finding communities is the twin of the act of making communities findable – and this is where a resource like pm.org can help everyone.
> 
> That's what pm.org does, right? It shows anyone looking for a group where the active groups are?

Yes, it does the basics, but could do so much more – like showing community status metadata clearly! :wink: 

> > Ooof. I can relate with this too. There seems to be a lot of "bystanderism" going on, it seems. Or a lack of interest in community building/care/sustainability in general? 😞
> 
> Broadly, there seems to be less interest in nerd meetups since covid. And broadly, Perl / Python / Golang appear to be less used, or used by people disinterested in physical meetups. (Which is fine. I get it. Jobs can be exhausting, maybe the last thing you want after 40 hours of coding is more coding / computer talk.)

:notes:  Yeah, yeah, yeah, the hype/excitement is gone, but there systems are still a-running!

> Again, don't let me dissuade you from extending our `perl_mongers.dtd`. Go for it! I'd love to find out that I'm wrong. Let me know how I can help.

If you'd like to, I'd love a sounding-board and co-brainstormer for this ticket! :sweat_smile:

**Comment by jhannah** [comment]
> I think Oslo.pm forgot to update this file after Marcus Ramberg stepped down.

Like this? #194

> I don't _need_ to be on the list. I was just curious about what was in the archives, but was blocked from looking at them as I wasn't a list member. I had forgotten that this one is only for group leaders.

I'll add you if you're the new group leader.

> (On a side note: I think groups should be allowed to assign more members, so they aren't forced into creating another burnout-prone SPOF due to "pm.org policy")

You can have as many group leaders as you want. Several groups have multiple `<tsar>` entries.

**Comment by sjn** [comment]
> > (On a side note: I think groups should be allowed to assign more members, so they aren't forced into creating another burnout-prone SPOF due to "pm.org policy")
> 
> You can have as many group leaders as you want. Several groups have multiple `<tsar>` entries.

Sweet! In that case, please keep Marcus on the list, until he says he wants to be removed! :)

**Comment by jhannah** [comment]
> Oh, and I guess some of the tooling in `bin/` may also need updating if the DTD/XML changes?

`bin/xml` is the only program that matters that I recall. I don't think it validates the `.dtd` file (did it ever?). But in principle, if you want to expand the XML spec, that's where you'd do it. 

Personally, I'm not worried about it. As long as `bin/xml` keeps running, I'm happy. I run it every time there's any group update, which is... 2-3 times a year nowadays? If you break it, no problem, I can fix it when I need to flush again.

Perl Mongers is down to [28 groups that have 1+ posts to their mailing list this year](https://x6.develooper.com/~mailman/list-last-posts). Presumably several more groups are active, using things other than MailMan? I want to empower you and others to do whatever you want to improve pm.org, but personally I need to throw my hours at landing a new job. Which shouldn't slow *you* down. ❤️

**Comment by sjn** [comment]
> Perl Mongers is down to [28 groups that have 1+ posts to their mailing list this year](https://x6.develooper.com/~mailman/list-last-posts). Presumably several more groups are active, using things other than MailMan? I want to empower you and others to do whatever you want to improve pm.org, but personally I need to throw my hours at landing a new job. Which shouldn't slow _you_ down. ❤️

I too am rather busy, and what I'm doing here is a "drive-by contribution", so that's why I'm asking for certain types of interaction. I'll slow down when the interaction slows down. :wink: 

With that said, those stats are useful! Can we integrate these somehow? (e.g. convert it to JSON and visualize it somehow in the UI?) If the code producing that file is in a repo somewhere, we could make it into JSON, for example?

Also, I just realized that these pages also can become more useful if we add a capability for group tzars to add links to "alternative communities" (e.g. meetup groups or "techies in the pub" events or whatever)...

What do you think?

**Comment by jhannah** [comment]
> With that said, those stats are useful! Can we integrate these somehow? (e.g. convert it to JSON and visualize it somehow in the UI?) If the code producing that file is in a repo somewhere, we could make it into JSON, for example?

MailMan is only one way groups might be communicating. It wouldn't be fair to such groups for pm.org to "shame" them for having low MailMan stats if they're active elsewhere. :) If a group is entirely inactive, we inactivate them until they find 1+ new leaders.

> Also, you've helped me realize these pages can become more useful if group tzars get the capability to add links to "alternative communities" (e.g. meetup groups or "techies in the pub" events or "nerd boards" or whatever)...

Several groups already use the XML to register their various social medias. The website puts little icons of those social medias on their group page at pm.org. e.g.:

    <web>http://perl.org.il/</web>
    <linkedin>http://www.linkedin.com/groups?gid=684</linkedin>
    <facebook>http://www.facebook.com/Israeli.Perl.Mongers</facebook>
    <googleplus>https://plus.google.com/105209189901369984525</googleplus>

**Comment by sjn** [comment]
> > With that said, those stats are useful! Can we integrate these somehow? (e.g. convert it to JSON and visualize it somehow in the UI?) If the code producing that file is in a repo somewhere, we could make it into JSON, for example?
> 
> MailMan is only one way groups might be communicating. It wouldn't be fair to such groups for pm.org to "shame" them for having low MailMan stats if they're active elsewhere. :) If a group is entirely inactive, we inactivate them until they find 1+ new leaders.

Ah, no. Why would you assume this is about _shaming_ anyone? :open_mouth: (WTF, seriously, man! :angry:)

I'm looking for ways to signal "volunteers needed" more explicitly. But sure, if someone insists at considering community health metrics, then mailing list activity _is_ a useful indicator. There is no shame in not volunteering, but if someone cares about their community/ecosystem/open source colleagues, then it is _good_ to be aware that activity produces a positive signal.

| Can I help?            | Not aware help is needed | Aware help is needed |
|---------------------------|----------------------------|-------------------------------------------|
| I have no tuits       | Nothing happens | Can spread word or recruit |
| I have some tuits  | Nothing happens | Can help a little organizing |
| I have lots of tuits | Nothing happens | Can help a lot organizing | 

If you add "shaming" to this table, you are basically blocking efforts that make more people aware. NOT A GOOD THING. :angry:

> > Also, you've helped me realize these pages can become more useful if group tzars get the capability to add links to "alternative communities" (e.g. meetup groups or "techies in the pub" events or "nerd boards" or whatever)...
> 
> Several groups already use the XML to register their various social medias. The website puts little icons of those social medias on their group page at pm.org. e.g.:
> 
> ```
> <web>http://perl.org.il/</web>
> <linkedin>http://www.linkedin.com/groups?gid=684</linkedin>
> <facebook>http://www.facebook.com/Israeli.Perl.Mongers</facebook>
> <googleplus>https://plus.google.com/105209189901369984525</googleplus>
> ```

Oh nice! Is this documented somewhere?

**Comment by jhannah** [comment]
> Oh nice! Is this documented somewhere?

https://www.pm.org/faq/hosting.html#social

**Comment by sjn** [comment]
> > Oh nice! Is this documented somewhere?
> 
> https://www.pm.org/faq/hosting.html#social

Right. This isn't reflected in [the DTD](https://github.com/perlorg/www.pm.org/blob/master/perl_mongers.dtd), though.

Also, after letting this sink in for a moment, "social media" isn't the same as "alternative communities", is it? :smile: 

I'm thinking of meetup and facebook groups that organize events. I get that there may be a concept overlap, though I think it would be clearer (and more useful?) if one can have a link for signing up/joining the member forums/communities from their SoMe contact points? Hmm.

**Comment by jhannah** [comment]
Yesterday @sjn gave me a 45m phone overview of SBOM and friends. Thanks! It all sounds very important for Perl and CPAN (cpanm, etc.). As far as Perl Mongers goes, here's where [the Slack thread](https://perlfoundation.slack.com/archives/C08DB50U7TK/p1741106101124279) stands?:

> **Me:** In my mostly-ignorance of all this, I'm not comfortable inventing a new SBOM-compatible-??? format for Perl Mongers that may or may not be helpful to someone somewhere someday. If someone finds an example, I could expand on it / audit against our groups XML.
> **sjn:** AFAICT, there are few-to-none who have thought substantially on this topic, so if you more inclined to wait, then you may need to exercise your patience. :confused:
With that said, the Sustainability WG I mentioned does have this topic in-scope, so if you have needs or use cases or experience-based opinions, then there's an opportunity to make a difference there. :slightly_smiling_face:
And with some careful TLC and luck, this may even end up as an ECMA standard :smile:
> **Me:** Is there some industry-standard tool I can run somewhere that injests SBOM files to tell me if it's happy with my file or not?
> **sjn:** might be worth looking at `cdxgen` for generating or manipulating CycloneDX files

So I think that's where this Issue stands. Next step is somebody (me?) could start playing with "`cdxgen` for generating or manipulating CycloneDX files."

**Comment by sjn** [comment]
Just for adding some clarity; The SBOMs (Software Bill of Materials) are meant for keeping track of metadata related to different aspects of a software product, component or project.

Think "dependency graph", "contact information", "repository", "license", etc.

Sustainability and community-related metadata aren't well supported at this moment, but as I mentioned in the Slack thread, this may change eventually – hopefully within the next year or so.

In other words, producing an SBOM today is very unlikely to produce anything useful for this particular case, as the tooling follows the specification, and the specification for this type of metadata isn't done yet.

So while the CycloneDX Sustainability WG tries to land this specification, we might as well do the thinking ourselves (and then make sure we communicate what we learn/discover to the folks who work on these specs) 😁

**Comment by jhannah** [comment]
March 4th in Perl Slack #cpan-security Todd Rinaldo wrote:
it looks like the BOM is based on https://cyclonedx.org/schema/bom-1.5.schema.json. The schema we have is rather large and I'm not sure if that'd be helpful to you.
here's a smaller mock example:
```
{
  "bomFormat": "CycloneDX",
  "specVersion": "1.5",
  "serialNumber": "urn:uuid:123e4567-e89b-12d3-a456-426614174000",
  "version": 1,
  "metadata": {
    "timestamp": "2025-03-04T12:00:00Z",
    "tools": [
      {
        "vendor": "WebPros",
        "name": "cPanel BOM Generator",
        "version": "1.0.0"
      }
    ],
    "authors": [
      {
        "name": "John Doe",
        "email": "johndoe@example.com"
      }
    ],
    "component": {
      "type": "application",
      "name": "cPanel",
      "version": "110.0.1",
      "group": "com.webpros",
      "purl": "pkg:maven/com.webpros/cpanel@110.0.1",
      "licenses": [
        {
          "license": {
            "id": "Proprietary"
          }
        }
      ],
      "hashes": [
        {
          "alg": "SHA-256",
          "content": "abcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890"
        }
      ]
    }
  },
  "components": [
    {
      "type": "library",
      "name": "Perl",
      "version": "5.38.0",
      "group": "org.perl",
      "purl": "pkg:maven/org.perl/perl@5.38.0",
      "licenses": [
        {
          "license": {
            "id": "Artistic-2.0"
          }
        }
      ]
    },
    {
      "type": "library",
      "name": "Playwright",
      "version": "1.40.0",
      "group": "com.microsoft",
      "purl": "pkg:npm/microsoft/playwright@1.40.0",
      "licenses": [
        {
          "license": {
            "id": "Apache-2.0"
          }
        }
      ]
    }
  ],
  "dependencies": [
    {
      "ref": "pkg:maven/com.webpros/cpanel@110.0.1",
      "dependsOn": [
        "pkg:maven/org.perl/perl@5.38.0",
        "pkg:npm/microsoft/playwright@1.40.0"
      ]
    }
  ]
}
```
[Slack thread](https://perlfoundation.slack.com/archives/C05KMAQKY6Q/p1741132693516179)

**Comment by jhannah** [comment]
On April 10, @sjn and I had a short call discussing gDoc "OSS Sustainabillity Use Cases and Requirements". Salve said it might be useful for me to give my 2¢ regarding imagining how Perl Mongers (and other user groups) would fit a record of "status / what each group needs" into that specification before they release v1.0 of that document. I haven't gotten around to that yet, but hope to... soon?
